src/Java.Runtime.Environment/Java.Interop/MonoRuntimeValueManager.cs

```diff
@@ -393,7 +388,7 @@ partial class JreNativeMethods {
 		internal static extern int java_interop_gc_bridge_set_current_once (IntPtr bridge);
 
 		[DllImport (JavaInteropLib, CallingConvention=CallingConvention.Cdecl)]
-		internal static extern int java_interop_gc_bridge_register_hooks (IntPtr bridge, GCBridgeUseWeakReferenceKind weak_ref_kind);
+		internal static extern int java_interop_gc_bridge_register_hooks (IntPtr bridge);
```

Comment:
This change introduces a "mismatch" between the native side and the managed side.  (Native side has an additional `[[maybe_unused]] int weak_ref_kind` parameter.)  This is nominally "fine" because it's C calling convention, but if we ever accidentally-on-purpose change the calling convention, the difference in number of parameters could cause stack corruption.

This should instead be:

```csharp
internal static extern int java_interop_gc_bridge_register_hooks (IntPtr bridge, int must_be_zero);
```

Reply:
That was unintentional. I initially kept the parameter as unused and then changed my mind to remove it…